### DISCUSSION TITLE
Doesn't tensorrt support one model, all resolutions?

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#90] [General]
I've so far made 5 or 7 different resolutions for 2 models, and so far it's 23GB. Seeing as it said "bigger range, less speed". Would further "research" from nvidia's side allow models to be as custom resolution adaptive as regular safetensors?

**Comment by contentis** [comment]
Currently, the model weights must be stored alongside the engines for TensorRT to work correctly. I'm working on a way that we can directly use the weights from the torch checkpoint. This should bring the size of an engine down to <20mb. 
This will take some time though...

You can already build with as large as a range as you want (given your GPU has enough VRAM). https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT/discussions/28
